nab/detectors/htmjava/src/main/java/nab/detectors/htmjava/HTMModel.java

```diff
@@ -138,9 +144,13 @@ public Parameters getSpatialPoolerParams(JsonNode modelParams) {
         if (spParams.has("synPermActiveInc")) {
             p.set(KEY.SYN_PERM_ACTIVE_INC, spParams.get("synPermActiveInc").asDouble());
         }
-        if (spParams.has("seed")) {
-            p.set(KEY.SEED, spParams.get("seed").asInt());
-        }
+        // if (spParams.has("seed")) {
+        //     p.set(KEY.SEED, spParams.get("seed").asInt());
+        // }
+        // For some seeds, htm.java throws
+        // java.lang.ArrayIndexOutOfBoundsException: 1017
+        //    at org.numenta.nupic.Connections.computeActivity(Connections.java:1199)
+        p.set(KEY.SEED, 42);
```

Comment:
Sure, I can change it to a TODO comment linking to the github issue.

Yeah, it's weird that we're passing in parameters that have multiple seeds while htm.java only uses one seed. The same thing happens with the column dimensions. I think a future PR could do some cleanup here.